src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
 }
+
+:root {
+  --blue_link_color: #00acdc;
+}
+
+.header {
+  padding-right: 50px;
+  position: relative;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav__item {
+  list-style-type: none;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.moyo__icon {
+  position: absolute;
+  left: 50px;
+  top: 10px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav_link {
+  display: block;
+  font-size: 12px;
+  line-height: 60px;
```

Comment:
use paddings instead

Reply:
in readme file there are some requirements  and it forbid to use any paddings for these elements